src/bmi_lgar.cxx

```diff
@@ -17,6 +17,14 @@
 // default verbosity is set to 'none' other option 'high' or 'low' needs to be specified in the config file
 string verbosity="none";
 
+/**
+ * @brief Delete dynamic arrays allocated in Initialize() and held by this object
+ * 
+ */
+BmiLGAR::~BmiLGAR(){
+  if( giuh_ordinates != nullptr ) delete [] giuh_ordinates;
+  if( giuh_runoff_queue != nullptr ) delete [] giuh_runoff_queue;
```

Comment:
C++ nitpick: `delete nullptr` is safe, there's no need to manually check.
```suggestion
  delete [] giuh_ordinates;
  delete [] giuh_runoff_queue;
```

Reply:
This applies all the way down the PR

Reply:
addressed in [d52d6f0](https://github.com/NOAA-OWP/LGAR-C/pull/28/commits/d52d6f0dd041b4c5d90004a3ff8285cfc28afd6c)